runtime/src/accounts_db.rs

```diff
@@ -40,7 +40,7 @@ use std::sync::atomic::{AtomicUsize, Ordering};
 use std::sync::{Arc, RwLock};
 use sys_info;
 
-const ACCOUNT_DATA_FILE_SIZE: u64 = 16 * 1024 * 1024;
```

Comment:
this effectively dictates the maximum size of account.data, right?

Reply:
Yes.